tests/tests.rs

```diff
@@ -1607,6 +1607,32 @@ fn regression_391() {
     assert_eq!(lines, "bar.py\n");
 }
 
+// See: https://github.com/BurntSushi/ripgrep/issues/451
+#[test]
+fn regression_451_only_matching() {
+    let wd = WorkDir::new("regression_451_only_matching");
+    let path = "digits.txt";
+    wd.create(path, "1 2 3\n123\n");
+
+    let mut cmd = wd.command();
+    cmd.arg("\\d").arg(path)
+        .arg("--only-matching")
+        .arg("--color=never")
```

Comment:
Assume you have changed the default colors of found result. Then all test will become invalidated.